csrankings-9.csv

```diff
@@ -1387,6 +1388,7 @@ Yaochun Shen,University of Liverpool,https://liverpool.ac.uk/electrical-engineer
 Yaohang Li,Old Dominion University,http://www.cs.odu.edu/~yaohang,NOSCHOLARPAGE
 Yaohui Jin,Shanghai Jiao Tong University,http://front.sjtu.edu.cn/~jinyh/index.html,H_7_oVcAAAAJ
 Yaoliang Yu,University of Waterloo,https://cs.uwaterloo.ca/~y328yu,zbXIQMsAAAAJ
+Yaonan Wang,Hunan University,http://eeit.hnu.edu.cn/info/1277/4490.htm,NOSCHOLARPAGE

```

Comment:
Not in CS; not clearly eligible for inclusion.

Reply:
The same as the first one.